packages/autorest.typescript/test/rlcIntegration/generated/bodyComplexRest/src/bodyComplexRestClient.ts

```diff
@@ -5,13 +5,19 @@ import { getClient, ClientOptions } from "@azure-rest/core-client";
 import { logger } from "./logger";
 import { BodyComplexRestClient } from "./clientDefinitions";
 
+export interface BodyComplexRestClientOptions extends ClientOptions {
+  apiVersion?: string;
+}
+
 /**
  * Initialize a new instance of `BodyComplexRestClient`
- * @param options - the parameter for all optional parameters
+ * @param {
+ *     apiVersion = "2016-02-29", ...options} - the parameter for all optional parameters
```

Comment:
thb I would keep the same @param description we had. This shouldn't impact the public API. The fat that we are destructuring internally is an implementaion detail

Reply:
updated.